src/app/error/components/500.tsx

```diff
@@ -54,28 +24,49 @@ const Error500 = () => {
           textAlign: "center",
         }}
       >
-        <BoxWrapper>
-          <Typography variant="h1">500</Typography>
+        <Styled.BoxWrapper>
           <Typography
-            variant="h5"
-            sx={{ mb: 1, fontSize: "1.5rem !important" }}
+            variant="h1"
+            sx={{
+              mb: 2.5,
+              fontSize: { md: "2rem !important", xs: "1.5rem !important" },
+            }}
           >
-            Internal server error 👨🏻‍💻
+            Error del servidor 👨🏻‍💻
```

Comment:
I prefer to avoid the word "error" here, also, not everyone know what a "servidor" is.
I suggest the following text:
> No se ha podido procesar tu petición

Also, I would remove the icon at the end of this text.